api/controllers/orderController.js

```diff
@@ -0,0 +1,42 @@
+const {orderService, cartService} = require("../services");
+const {catchAsync} = require("../utils/error");
+
+const getOrderList = catchAsync (async(req, res) => {
+
+        // const userId = req.user.id
+        const userId = 9;
+        const getOrderList = await orderService.getOrderList(userId);
+        if(getOrderList){
+                return res.status(201).json({getOrderList})
+        }else{
+                res.status(201).json({message:"포인트가 충전되었습니다."})
+        }
```

Comment:
```suggestion
        if(getOrderList){
                return res.status(201).json({getOrderList})
        }
        
        res.status(201).json({message:"포인트가 충전되었습니다."})
```
일단 else 없이 이렇게 써주시는게 낫긴한데, 해당 method 및 api 기능이 getOrderList인데 포인트가 충전되었습니다 는 뭐때문에 필요한거죠?

Reply:
잔여 포인트보다 결제해야 할 금액이 부족하면 주문페이지 조회 시 포인트가 다시 900000으로 충전되는 기능입니다.